packs/spell-effects/spell-effect-sages-curse-failure.json

```diff
@@ -0,0 +1,60 @@
+{
+    "_id": "wzmPGA1HTgNvvKmp",
+    "img": "systems/pf2e/icons/spells/savants-curse.webp",
+    "name": "Spell Effect: Sage's Curse (Failure)",
+    "system": {
+        "description": {
+            "value": "<p>The target must roll twice and use the worse result whenever attempting an Arcana, Lore, Occultism, or Society check.</p>"
```

Comment:
The reason I didn't do it is because I didn't see a rollable Recall Knowledge macro in the system to predicate that AdjustDegreeOfSuccess off of. Am I wrong about that?

Reply:
> Add a ChoiceSet for failure vs critical failure and add the AdjustDegreeOfSuccess for the crit fail effect.

done